packages/nimble-components/src/rich-text/editor/index.ts

```diff
@@ -475,6 +566,9 @@ export class RichTextEditor extends RichText implements ErrorPattern {
             'aria-disabled',
             this.disabled ? 'true' : 'false'
         );
+        if (this.disabled) {
+            this.mentionListBox?.close();
+        }
```

Comment:
This is surprising to me... What is the case where the editor is not disabled and we want to keep the mentionListBox open? MAke sure to comment abouut why this is significant.

Reply:
Removed the condition check and just closed the popup when disabled